api/wasm/README.md

```diff
@@ -0,0 +1,42 @@
+Envoy hooks:
```

Comment:
Let's wait for envoyproxy/envoy-wasm#516 (which is ready now) and remove this directory.

Reply:
I will merge that in and ping this thread.

Reply:
merged